examples/tcprtt_sockops/tcprtt_sockops.h

```diff
@@ -0,0 +1,45 @@
+#ifndef TCPRTT_SOCKOPS_H
```

Comment:
Is there a need to split the C across two files? I would find it easier to follow if we just had the .c.